docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -77,17 +80,88 @@ The following inspectors are defined:
 which always produces absolute URIs for incoming request based on the request URI and the `Host`-header of
 the request. You can customize this behavior by bringing a custom instance of `DefaultHostInfo` into scope.
 
+@@@
+
+@@@ div { .group-java }
+
+akka-http has a testkit that provides a convenient way of testing your routes with JUnit. It allows
+running requests against a route (without hitting the network) and provides means to assert against
+response properties in a compact way.
```

Comment:
I suggest this section is removed and we use the same introduction as the Scala version. The following text would make sense inside the `## Usage` section defined after the intro.